packages/react-composites/src/composites/CallComposite/pages/TransferPage.tsx

```diff
@@ -0,0 +1,225 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import {
+  IPersonaStyleProps,
+  IPersonaStyles,
+  IStyle,
+  IStyleFunctionOrObject,
+  Spinner,
+  SpinnerSize,
+  Stack,
+  Text,
+  mergeStyles
+} from '@fluentui/react';
+import { ErrorBar, OnRenderAvatarCallback } from '@internal/react-components';
+import React, { useCallback, useLayoutEffect, useMemo, useRef, useState } from 'react';
+import { useLocale } from '../../localization';
+import { CallArrangement } from '../components/CallArrangement';
+import { usePropsFor } from '../hooks/usePropsFor';
+import { useSelector } from '../hooks/useSelector';
+import { getRemoteParticipants } from '../selectors/baseSelectors';
+import { toFlatCommunicationIdentifier } from '@internal/acs-ui-common';
+import { AvatarPersona, AvatarPersonaDataCallback } from '../../common/AvatarPersona';
+/* @conditional-compile-remove(call-transfer) */
+import { getTransferCall } from '../selectors/baseSelectors';
+import { reduceCallControlsForMobile } from '../utils';
+import { LobbyPageProps } from './LobbyPage';
+
+/* @conditional-compile-remove(call-transfer) */
+type TransferPageState = 'transferring' | 'connecting';
+
+/**
+ * @private
+ */
+export const TransferPage = (
+  props: LobbyPageProps & {
+    /** Render component function to replace the default Persona Icon representing the transferor or transfer target. */
+    onRenderAvatar?: OnRenderAvatarCallback;
+    /** Callback function that can be used to provide custom data to Persona Icon rendered */
+    onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
+  }
+): JSX.Element => {
+  const errorBarProps = usePropsFor(ErrorBar);
+  const strings = useLocale().strings.call;
+  const remoteParticipants = useSelector(getRemoteParticipants);
+  /* @conditional-compile-remove(call-transfer) */
+  const transferCall = useSelector(getTransferCall);
+
+  // Reduce the controls shown when mobile view is enabled.
+  const callControlOptions = props.mobileView
+    ? reduceCallControlsForMobile(props.options?.callControls)
+    : props.options?.callControls;
+
+  /* @conditional-compile-remove(call-transfer) */
+  const pageState: TransferPageState = useMemo(() => {
+    if (transferCall !== undefined) {
+      if (['Ringing', 'Connected'].includes(transferCall.state)) {
+        return 'connecting';
```

Comment:
is this when we are being connected to the 'transfer' that is going to transfer us?

Reply:
The accepted transfer call that is already in CallClientState is in the state 'Ringing' or 'Connected' state. That is when the transfer target actually sees our call so that is when we are showing that we are in the 'connecting' page state as shown in this Figma.
![image](https://github.com/Azure/communication-ui-library/assets/79475487/e0563333-7aaa-490b-9aed-5135eaaccc45)
Maybe the page state 'connecting' is easily confused with the call state. Will try think of a better name.

Reply:
Good question. I changed the pageState variable to pageSubject. It is too easy to confuse with the call state. Page subject is basically who should be shown in the Transfer page, the transferor or the transfer target.